src/flutter/shell/platform/linux_embedded/surface/linuxes_surface_gl_eglstream.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
```

Comment:
Please rename both `drm` and `eglstream` all files to `drm_gbm` and `drm_eglstream`.

Reply:
What is the difference between "linuxes_surface_gl_drm.h" and `linuxes_surface_gl_eglstream.h`? Can you make it common?

Reply:
I renamed both `drm` and `eglstream` all files to `drm_gbm` and `drm_eglstream` and created `linuxes_surface_gl_drm.h` for the common code of `drm` and `eglstream`.